### DISCUSSION TITLE
Plot TSS enrichment and number of fragments in UMAP

### DISCUSSION THREAD

**Discussion by amibe1** [GreenleafLab/ArchR#1764] [Questions / Documentation]
Hi,

I have a question regrading creating some UMAP plots. I need to create plots of the cells in UMAP space and color each UMAP-plot by TSS enrichment and number of fragments. How can this be done in ArchR using the addUMAP() function?

Thank you!

**Comment by rcorces** [comment]
Please see the chapter on single-cell embeddings and read the function documentation for `plotEmbedding()`. TSS Enrichment and number of frags per cell are stored in `cellColData`
